### DISCUSSION TITLE
macOS: Installer window size is too small vertically

### DISCUSSION THREAD

**Discussion by chrisrimple** [jamulussoftware/jamulus#3455] [Bug Investigation]
Installing 3.11 on a 2021 MacBook Pro M1 Max running MacOS Sequoia 15.1.1, the installer window size is a bit too small vertically, which hides the names of the icons and requires vertical scrolling to read.  Monitor is an LG 43-inch 4k connected via USB-C.  Attached screenshot shows what I experienced.
![Screenshot 2024-12-29 at 10 24 17 AM](https://github.com/user-attachments/assets/d454356e-01fb-4a1d-be0e-5842ed3a6b52)

**Comment by ann0see** [comment]
I think that's due to having an enabled path bar and tabs. If they are disabled, everything should be sized correctly.

**Reply by chrisrimple** [reply]
I don't have tabs enabled for the Finder, but do have the Path Bar enabled.  Perhaps the icon locations in the Installer window should have a larger buffer from the bottom edge of the window?  (They seem to be bottom-aligned)

**Reply by ann0see** [reply]
Maybe we can tweak the sizing: https://github.com/jamulussoftware/jamulus/blob/fe88e3eb95fb733078cfc552e89d62ed4b8e8085/mac/deploy_mac.sh#L164

But then it would look wrong (potentially) if tabs are disabled.

**Reply by dcorson-ticino-com** [reply]
Why not just put those icons centered top-bottom in the window?

**Reply by ann0see** [reply]
They have absolute positions. I think you can also move the icons here: 
https://github.com/jamulussoftware/jamulus/blob/fe88e3eb95fb733078cfc552e89d62ed4b8e8085/mac/deploy_mac.sh#L168-L169
You can tweak the values in the script and try the produced artifact.
